examples/ingress_canary_and_auth.ipynb

```diff
@@ -0,0 +1,1088 @@
+    "\n",
+    "!juju deploy istio-gateway istio-ingressgateway --trust --channel 1.5/stable\n",
+    "!juju deploy istio-pilot --trust --config default-gateway=$gateway_name --channel 1.5/stable\n",
+    "# !juju deploy istio-gateway istio-ingressgateway --trust --config kind=ingress\n",
+    "# !juju deploy istio-pilot --trust --config default-gateway=$gateway_name\n",
+    "\n",
+    "!juju relate istio-pilot:istio-pilot istio-ingressgateway:istio-pilot\n",
+    "\n",
+    "# Note: $juju_model_name here in the istio-gateway is the namespace that the GATEWAY \n",
+    "# is deployed to, which in our example happens to be the same as the seldon deployment\n",
+    "# but in practice could be different.\n",
+    "!juju deploy seldon-core seldon-controller-manager --config istio-gateway=$juju_model_name/$gateway_name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f253399-4b12-439f-af39-9bd7c8e1fe30",
+   "metadata": {},
+   "source": [
+    "Wait for everything to deploy and settle, then get the gateway IP"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "cf324f5b-f05d-4571-a788-b3bda95f80a1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DEBUG:root:istio-ingressgateway/2 workload status is waiting since 2022-02-28 19:18:05+00:00\n",
+      "DEBUG:root:istio-pilot/2 workload status is maintenance since 2022-02-28 19:18:07+00:00\n",
+      "DEBUG:root:istio-pilot/2 juju agent status is executing since 2022-02-28 19:18:06+00:00\n",
+      "DEBUG:root:istio-ingressgateway/2 workload status is waiting since 2022-02-28 19:18:05+00:00\n",
+      "DEBUG:root:istio-ingressgateway/2 workload status is waiting since 2022-02-28 19:18:12+00:00\n",
+      "DEBUG:root:istio-pilot/2 workload status is maintenance since 2022-02-28 19:18:28+00:00\n",
+      "INFO:root:All units idle since 2022-02-28 19:18:33.363252Z (dex-auth/5, istio-ingressgateway/2, istio-pilot/2, oidc-gatekeeper/1, seldon-controller-manager/1)\n",
+      "DEBUG:root:dex-auth is lead by dex-auth/5\n",
+      "DEBUG:root:istio-ingressgateway is lead by istio-ingressgateway/2\n",
+      "DEBUG:root:istio-pilot is lead by istio-pilot/2\n",
+      "DEBUG:root:oidc-gatekeeper is lead by oidc-gatekeeper/1\n",
+      "DEBUG:root:seldon-controller-manager is lead by seldon-controller-manager/1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# sudo snap install juju-wait\n",
+    "!juju wait -vw"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "e8f22c36-452f-4d2b-9b03-0d5380fb2dac",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Our Istio ingressgateway ip: 10.64.140.43\n"
+     ]
+    }
+   ],
+   "source": [
+    "gateway_ip=!kubectl -n $juju_model_name get svc istio-ingressgateway -o yaml -o jsonpath='{.status.loadBalancer.ingress[0].ip}'\n",
```

Comment:
s/istio-ingressgateway/istio-ingressgateway-workload due to recent [changes](https://github.com/canonical/istio-operators/pull/72) on istio-gateway